Cakefile

```diff
@@ -434,3 +435,17 @@ task 'test:browser', 'run the test suite against the merged browser script', ->
   (-> eval source).call result
   testResults = runTests result.CoffeeScript
   process.exit 1 unless testResults
+
+task 'test:webpack', 'run webpack test', ->
+  args = [
+    "./node_modules/webpack/bin/webpack.js"
+    "--entry=./lib/coffeescript/browser.js", 
+    "--output-path=#{os.tmpdir()}",
+    "--output-filename=coffeescript.js"
+  ]
+
+  console.log 'webpack', args.slice(1).join(' ')
+
+  spawnNodeProcess args, 'both', (status) ->
+    console.log "webpack exits with #{status}"
```

Comment:
In the context of the test file, this should be `eq status, 0` (assuming you’re checking for a zero exit status).